### DISCUSSION TITLE
Adjusting CSGO CMD Parameters

### DISCUSSION THREAD

**Discussion by styxadmin** [GameServerManagers/LinuxGSM#3544] [Q&A] [answered]
I am hosting a CSGO Server. The server is live and I am able to join it.

What I cannot seem to determine from the documentation, is how to adjust the Command Line Parameters. 

Below is the option I speak of.

![image](https://user-images.githubusercontent.com/50232595/131326956-500d5323-b1e8-4899-b68f-3a1f4b8413f3.png)

How do I adjust the parameters ?

**Comment by visdmin** [comment, marked as answer]
Hi,

The following path from your `lgsm` installation directory `[installation dir]/lgsm/config-lgsm/csgoserver/` contains configuration files for your `csgo` server.

You can find the following files there:
- _default.cfg 
- common.cfg
- `csgoserver.cfg`
- secrets-common.cfg
- secrets-csgoserver.cfg

File: `csgoserver.cfg` has the start up command inside.
(these `startparameters` are just example and can be outdated, but you will find yours that are up-to-date in your own config file)
```
## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
startparameters="-game csgo -usercon -strictportbind -ip ${ip} -port ${port} +clientport ${clientport} +tv_port ${sourcetvport} +sv_setsteamaccount ${gslt} -tickrate ${tickrate} +map ${defaultmap} +servercfgfile ${servercfg} -maxplayers_override ${maxplayers} +mapgroup ${mapgroup} +game_type ${gametype} +game_mode ${gamemode} +host_workshop_collection ${wscollectionid} +workshop_start_map ${wsstartmap} -authkey ${wsapikey} -nobreakpad"
```

There you can manually change the start up parameters and add your own too !

For more information you can check the lgsm documentation here:
https://docs.linuxgsm.com/configuration/game-server-config
You can find more information about the config files, and how they work.

Here is link to Valve`s documentation with information about the different possible start-up parameters:
https://developer.valvesoftware.com/wiki/Command_Line_Options#Command-line_parameters_2

**Reply by styxadmin** [reply]
Thank you! This is precisely what I was looking for

**Reply by visdmin** [reply]
Your welcome, happy to help !

**Reply by visdmin** [reply]
@styxadmin 
If my answer satisfied your question, you could maybe mark this question status as `answered` 👍🏼
